hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java

```diff
@@ -185,22 +185,22 @@
  *  &lt;/property&gt;
  *  </code></pre>
  *
- * <p>When <tt>conf.get("tempdir")</tt> is called, then <tt>${<i>basedir</i>}</tt>
+ * <p>When <code>conf.get("tempdir")</code> is called, then <code>${<i>basedir</i>}</code>
```

Comment:
there's also the {@code } clause, but given search and replace is easier. your PR is better